src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/leftcurly/InputLeftCurlyEolSwitch.java

```diff
@@ -0,0 +1,60 @@
+package com.puppycrawl.tools.checkstyle.checks.blocks.leftcurly;
+/**
+  * This test-input is intended to be checked using following configuration:
+  *
+  * option = EOL
+  *
+  */
+public class InputLeftCurlyEolSwitch {
+
+    public void doStuff() {
+        int x = 1;
+        switch (x) {
+            case 0:
+            { // warn
+                break;
+            }
+            case (1+0):
+            { // warn
+                break;
```

Comment:
please add:
```
            case (1
                       +0):
            { // warn
```
eol should work file on wrapped code too.

As we have a Check in 3 modes, it is kind of 3 Checks in one.
It would be awesome to have such input files the same between all of modes with difference only in comments over class and inline comments.

Reply:
done

Reply:
Please recheck what is pushed, I still see only `case (1+0):` without wrapping

Reply:
It is here: https://github.com/checkstyle/checkstyle/pull/6025/files#diff-621e3fe826f56a7dc6de706a178a9365R37